### DISCUSSION TITLE
Add field edit_history_tweet_ids to Tweet object

### DISCUSSION THREAD

**Discussion by lewoudar** [tweepy/tweepy#1983] [Ideas]
Hello folks,
I am starting with the library and I am very satisfied with my use so far. So thanks to the team behind it!

Also, recently (29 September) Twitter had a `edit_history_tweet_ids` field to the tweet data. It is a default field that contains all the tweet ids representing the different versions of the tweet. More information can be found [here](https://developer.twitter.com/en/docs/twitter-api/fields). 

It will be good to have it on the Tweet object :)

**Comment by Harmon758** [comment]
It's already been added, with https://github.com/tweepy/tweepy/commit/73c05d051fc14b00271067f005ca42f2b68304c9, to the development version on the master branch.

**Reply by lewoudar** [reply]
aww Thanks!
